plugins/english/scribblehub.ts

```diff
@@ -74,23 +74,20 @@ class ScribbleHubPlugin implements Plugin.PluginBase {
         return this.parseNovels(loadedCheerio);
     }
 
-    async parseNovelAndChapters(novelUrl: string): Promise<Plugin.SourceNovel> {
-        const url = novelUrl;
-        const result = await fetchApi(url);
+    async parseNovel(novelPath: string): Promise<Plugin.SourceNovel> {
+        const result = await fetchApi(this.site + novelPath);
```

Comment:
Can you test this on webtest? I have to set `User-Agent` to undefined to bypass cf

Reply:
yes